code/object/object.cpp

```diff
@@ -926,15 +927,13 @@ void obj_move_call_physics(object *objp, float frametime)
 			// then reset the flag and don't move the object.
             if (MULTIPLAYER_MASTER && (objp->flags[Object::Object_Flags::Just_updated])) {
 				objp->flags.remove(Object::Object_Flags::Just_updated);
-				goto obj_maybe_fire;
```

Comment:
Love that you were able to get rid of a `goto` statement.  Those are just plain confusing.